.github/workflows/deploy.yml

```diff
@@ -24,13 +24,8 @@ jobs:
           path: node_modules
           # Don't share with other workflows because we are using `--production=false`
           key: dev-dependencies-${{ hashFiles('**/yarn.lock') }}
-      # We need to create a .npmrc with the fontawesome token
       - name: Setup fontawesome dependencies
-        run: |
-          cat > .npmrc <<EOF
-          @fortawesome:registry=https://npm.fontawesome.com/
-          //npm.fontawesome.com/:_authToken=${{ secrets.FONTAWESOME_TOKEN_GITHUB }}
-          EOF
+        run: yarn config set -H npmScopes.fortawesome.npmAuthToken ${{ secrets.FONTAWESOME_TOKEN_GITHUB }}
```

Comment:
@coleEthos Do you know if we are allowed to rehost fontawesome in our own package registry? If so, we could do that and get rid of this extra token and simplify things further.

Considering you can serve it from your own server as [described here](https://docs.fontawesome.com/web/setup/host-yourself/svg-js), I feel like this might be within the TOS.

Reply:
Could also remove the chance where we run out of bandwidth allowances or run into what happened recently where the charge failed.

Reply:
@tylermichael it seems like we could likely do that without violating TOS, it seems to me like something that counts as self hosting, even if their documentation doesn't describe how to implement what you're proposing.

Want me to email and check with support to confirm?

Reply:
Yea that might be best just to be safe. I'd be surprised if that goes against TOS.

I would mention that it would be hosted in a private NPM registry that requires an auth key etc.

Reply:
Okay emailing them now.

Reply:
@tylermichael I cc'd you but this was their response: 
> As long as you are not providing the pro assets to non-seat users you should be good.